rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1328,18 +1328,36 @@ def test_upgrade_db_23_to_24(user_data_dir):  # pylint: disable=unused-argument
     """
     msg_aggregator = MessagesAggregator()
     _use_prepared_db(user_data_dir, 'v23_rotkehlchen.db')
+    db = _init_db_with_target_version(
+        target_version=23,
+        user_data_dir=user_data_dir,
+        msg_aggregator=msg_aggregator,
+    )
+    cursor = db.conn.cursor()
+
+    # Checks before migration
+    assert cursor.execute(
+        'SELECT COUNT(*) from used_query_ranges WHERE name LIKE "adex_events%";',
+    ).fetchone()[0] == 1
+    assert cursor.execute(
+        'SELECT COUNT(*) from adex_events;',
```

Comment:
nitpick: This fits in one line:

`assert cursor.execute('SELECT COUNT(*) from adex_events;').fetchone()[0] == 1`